Traceback (most recent call last):
  File "/home/alexandrasouverneva/Desktop/my-book-files/venv/lib64/python3.9/site-packages/jupyter_core/utils/__init__.py", line 154, in wrapped
    asyncio.get_running_loop()
RuntimeError: no running event loop

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/alexandrasouverneva/Desktop/my-book-files/venv/lib64/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/alexandrasouverneva/Desktop/my-book-files/venv/lib64/python3.9/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/alexandrasouverneva/Desktop/my-book-files/venv/lib64/python3.9/site-packages/jupyter_core/utils/__init__.py", line 158, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/lib64/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/alexandrasouverneva/Desktop/my-book-files/venv/lib64/python3.9/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/alexandrasouverneva/Desktop/my-book-files/venv/lib64/python3.9/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/alexandrasouverneva/Desktop/my-book-files/venv/lib64/python3.9/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
res = requests.get('https://pubchem.ncbi.nlm.nih.gov/rest/pug/compound/cid/1174/property/isomericsmiles/txt')
img = Draw.MolToImage( Chem.MolFromSmiles( res.text.rstrip() ), size=(150, 150) )
img
------------------

----- stderr -----
[10:36:22] SMILES Parse Error: syntax error while parsing: Status:
[10:36:22] SMILES Parse Error: check for mistakes around position 2:
[10:36:22] Status:
[10:36:22] ~^
[10:36:22] SMILES Parse Error: Failed parsing SMILES 'Status:' for input: 'Status:'
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[10], line 2[0m
[1;32m      1[0m res [38;5;241m=[39m requests[38;5;241m.[39mget([38;5;124m'[39m[38;5;124mhttps://pubchem.ncbi.nlm.nih.gov/rest/pug/compound/cid/1174/property/isomericsmiles/txt[39m[38;5;124m'[39m)
[0;32m----> 2[0m img [38;5;241m=[39m [43mDraw[49m[38;5;241;43m.[39;49m[43mMolToImage[49m[43m([49m[43m [49m[43mChem[49m[38;5;241;43m.[39;49m[43mMolFromSmiles[49m[43m([49m[43m [49m[43mres[49m[38;5;241;43m.[39;49m[43mtext[49m[38;5;241;43m.[39;49m[43mrstrip[49m[43m([49m[43m)[49m[43m [49m[43m)[49m[43m,[49m[43m [49m[43msize[49m[38;5;241;43m=[39;49m[43m([49m[38;5;241;43m150[39;49m[43m,[49m[43m [49m[38;5;241;43m150[39;49m[43m)[49m[43m [49m[43m)[49m
[1;32m      3[0m img

File [0;32m~/Desktop/my-book-files/venv/lib64/python3.9/site-packages/rdkit/Chem/Draw/__init__.py:101[0m, in [0;36mMolToImage[0;34m(mol, size, kekulize, wedgeBonds, fitImage, options, **kwargs)[0m
[1;32m     71[0m [38;5;250m[39m[38;5;124;03m"""Returns a PIL image containing a drawing of the molecule[39;00m
[1;32m     72[0m 
[1;32m     73[0m [38;5;124;03m    ARGUMENTS:[39;00m
[0;32m   (...)[0m
[1;32m     98[0m [38;5;124;03m      a PIL Image object[39;00m
[1;32m     99[0m [38;5;124;03m"""[39;00m
[1;32m    100[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m mol:
[0;32m--> 101[0m   [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m([38;5;124m'[39m[38;5;124mNull molecule provided[39m[38;5;124m'[39m)
[1;32m    102[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mhasattr[39m(rdMolDraw2D, [38;5;124m'[39m[38;5;124mMolDraw2DCairo[39m[38;5;124m'[39m):
[1;32m    103[0m   [38;5;28;01mraise[39;00m [38;5;167;01mRuntimeError[39;00m([38;5;124m"[39m[38;5;124mMolToImage requires that the RDKit be built with Cairo support[39m[38;5;124m"[39m)

[0;31mValueError[0m: Null molecule provided

